cmd/ketch/app_deploy.go

```diff
@@ -78,3 +80,18 @@ func newAppDeployCmd(cfg config, params *deploy.Services, configDefaultBuilder s
 	})
 	return cmd
 }
+
+func appDeploy(cmd *cobra.Command, options deploy.Options, params *deploy.Services) error {
+	var changeSet *deploy.ChangeSet
+	var err error
+	switch {
+	case validation.ValidateYamlFilename(options.AppName):
+		changeSet, err = options.GetChangeSetFromYaml(options.AppName)
```

Comment:
it would be great to change the description of `app deploy` to show that we can handle app.yaml

Reply:
Good call. I always forget about documentation/usage. I added some words at line 28.